CIPs/cip-0033.md

```diff
@@ -0,0 +1,164 @@
+---
+cip: 33
+title: Multi Currency Support
+author: Nadiem Sissouno (@sissnad)
+discussions-to: https://github.com/celo-org/celo-proposals/issues/125
+status: Draft
+type: Standards Track
+category: Ring 0
```

Comment:
nit: I think this is `Ring 1`. Seems like a core feature that won't cause chain halt if we mess something up

Reply:
Moving it to Ring 1 based on the mental model by @prestwich that describes the classification best:
```
classify by failure modes:
ring 0: halt the chain (liveness or safety violation), require manual intervention
ring 1: chain does bad things but keeps running
```

Given it's an upgrade to core contracts that doesn't halt the chain, it's a Ring 1. Will update the category and approve and merge the PR.